### PR TITLE
fix zap logger rotation issues

### PR DESCRIPTION
**Pull request by paulyufan2**
**Reason for Change**:

This PR is to fix log rotation issue that is crashed after windows nodes are restarted.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Comment by vipul-21** [comment]
Can we add more to the description about what the rotation issue is ?

File: cni/network/plugin/main.go [hunk 1]
```diff
@@ -295,15 +294,6 @@ func main() {
 		os.Exit(0)
 	}
 
-	log.SetName(name)
```
Comment (vipul-21) [inline cni/network/plugin/main.go:298]: These were added to support a known issue(incompatilbility of loggers between cns/cni) if i remember correctly ? Is that no longer valid ?

Comment (paulyufan2) [reply cni/network/plugin/main.go:298]: yes after we migrate all legacy logs to zap logger, we do not need this anymore and it is using legacy logs, trying to rotate log files but in the meantime, zap logger lumberjack is rotating log files as well, causing windows pods to be crashed and cannot get back. So this PR is to fix this issue.

Comment (vipul-21) [reply cni/network/plugin/main.go:298]: Let's run CNI release pipeline with this change too

Comment (paulyufan2) [reply cni/network/plugin/main.go:298]: yes it's running

File: cni/ipam/plugin/main.go [hunk 2]
```diff
@@ -23,17 +22,6 @@ func main() {
 	var config common.PluginConfig
 	config.Version = version
 
-	logDirectory := "" // Sets the current location as log directory
```
Comment (tamilmani1989) [inline cni/ipam/plugin/main.go:26]: did we validate ipam logs moved to zap

Comment (paulyufan2) [reply cni/ipam/plugin/main.go:26]: yes I did with this PR: https://github.com/Azure/azure-container-networking/pull/2299
Verified azure-ipam.log

**Review by tamilmani1989 — APPROVED** [review]

**Review by vipul-21 — APPROVED** [review]

**Review by behzad-mir — COMMENTED** [review]
plz make ssure we don't have any "github.com/Azure/azure-container-networking/log in the CNI and network related packages

**Review by behzad-mir — APPROVED** [review]

**Pull request merged**